### PR TITLE
Nuget cleanup

### PR DESCRIPTION
**Pull request by martijn00**

### CODE REVIEW

File: DSTV.Net/Extensions/TextReaderExtensions.cs [hunk 1]
```diff
@@ -96,6 +96,7 @@ internal static async Task<int> ParseInteger(this TextReader reader, ReaderConte
     /// <summary>
     ///     Parses an enum of type <see cref="TEnum" />
     /// </summary>
+    /// <typeparam name="TEnum"></typeparam>
```
Comment (tluijken) [inline DSTV.Net/Extensions/TextReaderExtensions.cs:99]: Als we m toevoegen, moeten we m eigenlijk ook vullen met wat zinnige informatie. Wat is de betekening van TEnum hier?

**Review by tluijken — CHANGES_REQUESTED** [review]
I'm getting issues on the IDisposable analyzer when building the project.

CSC : error CS9057: The analyzer assembly '\~/packages/idisposableanalyzers/4.0.6/analyzers/dotnet/cs/IDisposableAnalyzers.dll' references version '4.5.0.0' of the compiler, which is newer than the currently running version '4.4.0.0'. [\~/projects/baseflow/DSTVReader/DSTV.Net/DSTV.Net.csproj]

Do you know why? I have dotnet SDK 6.0.100, 6.0.116 and 7.0.105 installed on my machine.
I ran:
```shell
dotnet clean
dotnet restore
dotnet build
```

File: DSTV.Net/Implementations/HeaderReader.cs [hunk 2]
```diff
@@ -10,6 +10,7 @@ internal sealed class HeaderReader
     private const char Space = ' ';
     private const char Comment = '*';
 
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "MA0051:Method is too long", Justification = "<Pending>")]
```
[the remark this answers is not in the document]
Comment (martijn00) [reply DSTV.Net/Implementations/HeaderReader.cs]: Hmm not sure. Maybe just fix it later whenever anyone feels like it?

**Comment by martijn00** [comment]
@tluijken I don't know, for me it builds fine on mac and windows.

**Pull request merged**